### PR TITLE
migration: Add a case about network data transport

### PR DESCRIPTION
**Pull request by cliping**
VIRT-293885 - VM live migration - network data transport - UNIX+Proxy

Signed-off-by: cliping <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 migration.migration_uri.network_data_transport.unix_proxy
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 2bcb7e7460ac16c820f07d729780f9c30a5d8153
JOB LOG    : /var/lib/avocado/job-results/job-2022-07-07T11.03-2bcb7e7/job.log
 (1/4) type_specific.io-github-autotest-libvirt.migration.migration_uri.network_data_transport.unix_proxy.default.with_precopy.p2p: STARTED
 (1/4) type_specific.io-github-autotest-libvirt.migration.migration_uri.network_data_transport.unix_proxy.default.with_precopy.p2p: PASS (165.43 s)
 (2/4) type_specific.io-github-autotest-libvirt.migration.migration_uri.network_data_transport.unix_proxy.default.with_precopy.non_p2p: STARTED
 (2/4) type_specific.io-github-autotest-libvirt.migration.migration_uri.network_data_transport.unix_proxy.default.with_precopy.non_p2p: PASS (165.17 s)
 (3/4) type_specific.io-github-autotest-libvirt.migration.migration_uri.network_data_transport.unix_proxy.default.with_postcopy.p2p: STARTED
 (3/4) type_specific.io-github-autotest-libvirt.migration.migration_uri.network_data_transport.unix_proxy.default.with_postcopy.p2p: PASS (167.03 s)
 (4/4) type_specific.io-github-autotest-libvirt.migration.migration_uri.network_data_transport.unix_proxy.default.with_postcopy.non_p2p: STARTED
 (4/4) type_specific.io-github-autotest-libvirt.migration.migration_uri.network_data_transport.unix_proxy.default.with_postcopy.non_p2p: PASS (167.18 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2022-07-07T11.03-2bcb7e7/results.html
JOB TIME   : 667.27 s

**Comment by cliping** [comment]
Depend on: https://github.com/avocado-framework/avocado-vt/pull/3447

**Review by Yingshun — APPROVED** [review]

File: libvirt/tests/cfg/migration/migration_uri/migration_network_data_transport_unix_proxy.cfg [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        - default:
+            func_supported_since_libvirt_ver = (6, 6, 0)
+            transport_type = "unix_proxy"
+            desturi_socket_path = "/tmp/desturi-socket"
+            migrateuri_socket_path = "/var/lib/libvirt/qemu/migrateuri-socket"
+            disks_uri_socket_path = "/var/lib/libvirt/qemu/disks-uri-socket"
+            virsh_migrate_desturi = "qemu+unix:///system?socket=${desturi_socket_path}"
+            virsh_migrate_migrateuri = "unix://${migrateuri_socket_path}"
+            virsh_migrate_disks_uri = "unix://${disks_uri_socket_path}"
+            virsh_migrate_extra = "--migrateuri ${virsh_migrate_migrateuri} --bandwidth 20"
```
Comment (fangge1212) [inline libvirt/tests/cfg/migration/migration_uri/migration_network_data_transport_unix_proxy.cfg:47]: I see migrate_speed is set to 10 for precopy, while it is set to 20 here, which one is expected?

Comment (cliping) [reply libvirt/tests/cfg/migration/migration_uri/migration_network_data_transport_unix_proxy.cfg]: Removed '--bandwidth 20'. Thanks.

**Review by fangge1212 — APPROVED** [review]

**Pull request merged**